src/modules/75-cf/pages/onboarding/__test__/ValidatingYourFlag.test.tsx

```diff
@@ -163,9 +163,12 @@ const renderComponent = (): RenderResult =>
           identifier: 'xxx-xxx-xxx',
           type: 'server'
         }}
-        environmentIdentifier="foo-123-bar"
+        environmentIdentifier={'foo-123-bar'}
```

Comment:
Basic strings don't need parens

```suggestion
        environmentIdentifier="foo-123-bar"
```